src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+import java.time.LocalDateTime;
+
+public class Deadline extends Task {
+    private LocalDateTime deadline;
+    public Deadline(String in, LocalDateTime deadline) {
+        super(in);
+        this.deadline = deadline;
+    }
+    public String toString() {
+        String donez;
+        if (done) {
```

Comment:
"done" seems like a boolean value, maybe try "isDone"?

Reply:
boolean method can be named isDone() instead.